### DISCUSSION TITLE
AWS/Azure why is there a software charge?

### DISCUSSION THREAD

**Discussion by korzynski** [Security-Onion-Solutions/securityonion#7640] [General] [answered]
In the AWS marketplace AMI for Security Onion 2, the software charges are listed as $0.15/hr, plus the infrastructure cost. This amounts to over $1,300/year just for the software, but the AMI doesn't appear to include any support, services, or features that aren't included with the free download.

The Azure image has a software cost as well, which appears to range from as little as $0.028/hr to over $15/hr!

Why is there a software charge for instances in AWS and Azure? Is there some reason I would pay that additional hourly cost when I could just install the software on a VM at one of those providers for free?

**Comment by dougburks** [comment, marked as answer]
Our official cloud images will save you time and effort. The AWS and Azure charges help us to pay our significant AWS and Azure costs and also help us to continue to build and maintain Security Onion as a project.
